omegaconf/omegaconf.py

```diff
@@ -128,16 +158,16 @@ def env(key: str, default: Any = _EMPTY_MARKER_) -> Any:
             val = empty_config.resolve_parse_tree(parse_tree)
         except InterpolationKeyError as exc:
             raise InterpolationKeyError(
-                f"When attempting to resolve env variable '{key}', a node interpolation "
-                f"caused the following exception: {exc}. Node interpolations are not "
-                f"supported in environment variables: either remove them, or escape "
-                f"them to keep them as a strings."
+                f"When attempting to resolve expression `{expr}` in `oc.decode`, "
+                f"a node interpolation caused the following exception: {exc}. "
+                f"Node interpolations are not supported by `oc.decode`."
```

Comment:
Does it make sense to keep interpolations as strings instead of trying to resolve them here?

Reply:
Just to be clear, this specific quoted piece of code is outdated, because `oc.decode` can actually resolve interpolations (sorry about that, I should have pointed it out in the review notes).
See the second commit b29a019b9fa49e574652a8504b4b3fbaaea3f0df